apps/charterafrica/src/components/LongForm/LongForm.js

```diff
@@ -33,7 +33,7 @@ const LongForm = React.forwardRef(function LongForm(props, ref) {
         {content.map((c) => {
```

Comment:
using plurals for naming lists then while using the map method we can use the singular reference
e.g `contents.map((content) => { ...`

Reply:
💯 ... except our CMS has chosen to call the page contents `content` so we're stuck with that for now.

Reply:
What CMS are we using?

Reply:
[Payload CMS](https://payloadcms.com/)... Speaking of CMS, I'd also like to hear you views and experiences with various CMSes.